src/main/webapp/WEB-INF/jsp/contributions/privacy-policy.jsp

```diff
@@ -0,0 +1,39 @@
+<content:title>
+    <fmt:message key="privacy.policy" />
+</content:title>
+
+<content:section cssId="privacyPolicyPage">
+    <h2>🔒 <content:gettitle /></h2>
+    
+    <p>
+        This policy provides transparency on how the elimu.ai Community protects 
+        the privacy of its community members (contributors):
+    </p>
+    
+    <ol class="browser-default">
+        <li>
+            When a community contributor signs onto this website or the 
+            <a href="https://github.com/elimu-ai/crowdsource" target="_blank">Crowdsource</a> 
+            Android app for the first time, we store the anonymous provider ID (from 
+            Google/GitHub) in order to recognize the same contributor when 
+            she returns at a later point in time.
```

Comment:
nitpick: Should we consider: he/she here?

Reply:
> nitpick: Should we consider: he/she here?

@Keerthi4308 Fixed 🙂